src/prisma/schema.prisma

```diff
@@ -1956,6 +1962,7 @@ model FormField {
   options     Json?
   formId      String            @db.Uuid
   form        Form              @relation(fields: [formId], references: [id], onDelete: Cascade)
+  extraFields Json?
```

Comment:
i think we can remove this?

Reply:
We need this to store the required & hidden status of the proposal project profile. I am renaming this to `fieldConfig` otherwise this is a bit confusing